src/library/modules/Service.js

```diff
@@ -483,6 +483,26 @@ See Manifest File [manifest.json] under "background" > "scripts"
 		}
 	});
 	
+	/* On Chrome Storage Changed
+	Sync localStorage parts with Chrome Storage
+	Used for sync quests data on different machines
+	------------------------------------------*/
+	chrome.storage.onChanged.addListener(function(changes, namespace) {
+		for (var key in changes) {

```

Comment:
Is there a reason you're enumerating over all properties instead of doing something like
`if (namespace === 'sync' && changes.KC3QuestsData) {`
?